tools/sggoreview/tools.go

```diff
@@ -27,7 +27,8 @@ func PrepareCommand(ctx context.Context) error {
 	toolDir := sg.FromToolsDir(toolName)
 	binDir := filepath.Join(toolDir, version, "bin")
 	binary := filepath.Join(binDir, toolName)
-	hostOS := strings.Title(runtime.GOOS)
+	goos := []rune(runtime.GOOS)
+	hostOS := string(append([]rune{unicode.ToUpper(goos[0])}, goos[1:]...)) // capitalizes the first letter.
```

Comment:
If anyone knows of a better way to do this, let me know :sweat_smile:

Reply:
The documentation states:
>Deprecated: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead.

but I didn't want to add a dependency just for this.

Reply:
we have a helper function that gets copied around quite a bit - if you search for unicode.ToUpper you'll find it :smile:

Reply:
aight, I'll leave as is for now :sweat_smile: